lib/TraackrApi.php

```diff
@@ -92,6 +94,20 @@ public static function setCustomerKey($key) {
    } // End function setCustomerKey()
 
 
+   public static function setExtraHeaders($headers) {
```

Comment:
Perhaps an is_array() check? If we don't have strict type-checking on other params, no worries.